spec/active_stash/active_record_encryption_spec.rb

```diff
@@ -0,0 +1,79 @@
+require_relative "../support/user4"
+require_relative "../support/migrations/create_users4"
+
+require 'rspec/expectations'
+
+RSpec::Matchers.define :have_an_exact_index do |name|
+  match do |actual|
+    target = actual.find { |i| i.type == :exact }
+    !target.nil? && target.name == name
+  end
+end
```

Comment:
Are these matchers the same as in other tests? If so, we could probably move them into `spec/spec_helper.rb`.

Reply:
Yep good call, i'll move them over.